src/main/resources/lang/proxvc/en_US.lang

```diff
@@ -1,7 +1,10 @@
 gui.options.page.controls.category.proxvc=Voice Chat
 gui.options.page.audio.category.proxvc=Voice Chat
+gui.options.page.proxvc=Voice Chat
+gui.options.page.proxvc.category.input=Microphone Input
```

Comment:
these ids do not exist since you're registering the category/page with a wrong id.